pkgs/development/php-packages/sqlsrv/default.nix

```diff
@@ -12,7 +12,7 @@ buildPecl {
 
   meta = with lib; {
     description = "Microsoft Drivers for PHP for SQL Server";
-    license = licenses.mit;
+    license = licenses.bsd2;
```

Comment:
I guess this wasn't intentional to commit here?

Reply:
yes, i removed it, sorry.